packages/components/src/utils/property-checkers/check-one-of.ts

```diff
@@ -1,18 +1,23 @@
+import { EMPTY_VALUES } from './constants';
+
 export function checkOneOf<T extends { host: HTMLElement }>(
   component: T,
   prop: keyof T,
   possibleValues: readonly unknown[],
-  customMessage?: string,
 ) {
   const componentName = component.host.localName;
   const value = component[prop];
-  const defaultMessage = `The prop \`${String(
+
+  if (EMPTY_VALUES.some(v => v === value)) {
```

Comment:
Can you update this in all checkers?
```suggestion
  if (isValueEmpty(value)) {
```